CONTRIBUTING.md

```diff
@@ -58,6 +64,16 @@ To run the unit tests:
 (.env)$ python -m pytest
 ```
 
+To run the front-end tests:
+
+```bash
+$ cd /your/project/path
+$ npx cypress open
```

Comment:
We can safely assume that contributing use a virtual environment when working with Hedy. This can be replaced by simply stating `npx cypress open`. Did you add `cypress` to the requirements.txt as well?

Reply:
Yes, you're right. I changed that!